src/meta/api/src/schema_api_impl.rs

```diff
@@ -2431,6 +2431,38 @@ impl<KV: kvapi::KVApi<Error = MetaError> + ?Sized> SchemaApi for KV {
         Ok(table_name.unwrap().table_name)
     }
 
+    #[logcall::logcall("debug")]
+    #[minitrace::trace]
+    async fn list_tables_name_by_id(
+        &self,
+        table_id: Vec<MetaId>,
+    ) -> Result<Vec<String>, KVAppError> {
+        debug!(req :? =(&table_id); "SchemaApi: {}", func_name!());
+
+        let mut kv_keys = Vec::with_capacity(table_id.len());
+        for id in &table_id {
+            let k = TableIdToName { table_id: *id }.to_string_key();
+            kv_keys.push(k);
+        }
+
+        // Batch get all table-name by id
+        let seq_names = self.mget_kv(&kv_keys).await?;
+        let mut table_names = Vec::with_capacity(kv_keys.len());
+
+        for (i, seq_name_opt) in seq_names.iter().enumerate() {
+            if let Some(seq_name) = seq_name_opt {
+                let name_ident: DBIdTableName = deserialize_struct(&seq_name.data)?;
+                table_names.push(name_ident.table_name);
+            } else {
+                debug!(
```

Comment:
It's not meaningfu to add a `debug!` in `else` branch, when to use it?

Reply:
A user can be granted by immuteable db or table and muteable db.

like this:

grant select on system.* to a;
grant update on default.* to a;

Under muteable catalog can not find system db'id.

Reply:
Suggestions remove this debug, it's no use on production, and less use for development.

Reply:
Ok. Comment is better?

Reply:
Fixed.